karma.conf.js

```diff
@@ -46,6 +46,15 @@ const newWebpackConfig = Object.assign({}, webpackConfigProd, {
     }, {
       test: /\.svg$/,
       loader: 'url?limit=10000&mimetype=image/svg+xml',
+    }, {
+      test: /\.jpg$/,
+      loader: 'url?limit=10000&mimetype=image/jpeg',
+    }, {
+      test: /\.webm$/,
```

Comment:
If we had a small video wouldn't we want it inlined though? I feel like the benefits would be the same as for images. I doubt we'll have a video that small but none the less.